### ISSUE TITLE
WinXP checked/debug build BSODs during setup

### ISSUE THREAD

**Issue by WinFutureX** [86Box/86Box#2106]
**Describe the bug**
When installing the checked build of Windows XP, it will crash during installation. The free version installs just fine.

**To Reproduce**
1. Install XP as normal
2. See BSOD when rebooting after the text-mode portion
3. Reset the machine to continue setup
4. During the "installing devices" part, the OS will crash again

**Expected behavior**
XP should install properly without crashing.

**Screenshots**
BSOD during text-mode reboot:
![20220212-124539-872](https://user-images.githubusercontent.com/29031916/153713269-10812480-4fdc-400b-900e-8cc6089fac2d.png)

BSOD during GUI setup stage:
![20220212-131209-683](https://user-images.githubusercontent.com/29031916/153713281-64181058-0c91-4c8e-a041-c8c66a5302b7.png)

**Desktop (please complete the following information):**
 - OS: Win11 build 22000.493
 - 86Box version: Build 3542
 - Build information: Old dynarec, debug, 32 bit

**Additional context**
Also tried installing WinXP checked/debug in VMware and QEMU, both install fine.

**Comment by richardg867** [comment]
XP Checked is known to hit assertion failures, which cause a BSOD if WinDbg is not attached (not possible until proper serial port emulation is written), on the following drivers:

* `usbuhci` (UHCI USB controller)
* `agp440` (Intel chipset AGP) - untested after the last-minute GART implementation for v3.0
* `es1371` (Ensoniq AudioPCI)

As a temporary workaround, I would suggest using a VIA chipset machine to avoid `agp440`, with USB disabled in the BIOS to avoid `usbuhci`, and an ISA sound card to avoid `es1371mp`.

**Comment by WinFutureX** [comment]
I just checked Device Manager on a working WinXP VPC2007 instance which is the same machine I was using and since there is no AGP controller, that means agp440.sys was not loaded and thus was not the cause of the BSOD. I also used a Crystal CS4237B sound card, so that probably means the crash occurred because of the USB controller.

By the way, is it possible to create a variant of the PIIX4(E) without USB both as a partial workaround as well as more closely mirror VPC2007's hardware config?
